plugins/commands/reporter/src/main/kotlin/ReporterCommand.kt

```diff
@@ -204,10 +205,10 @@ class ReporterCommand : OrtCommand(
             ortResult = ortResult.replaceConfig(config)
         }
 
-        val resolutionProvider =
-            ortResult.resolvedConfiguration.resolutions.takeUnless { refreshResolutions }?.let { resolutions ->
-                DefaultResolutionProvider(resolutions)
-            } ?: DefaultResolutionProvider.create(ortResult, resolutionsFile)
+        if (refreshResolutions) {
```

Comment:
Condition should be `refreshResolutions || ortResult.resolvedConfiguration.resolutions == null` to also resolve resolutions if they have not been resolved before (becuase the evaluator did not run).

Reply:
Nice catch.